src/index.js

```diff
@@ -24,8 +24,12 @@ const checkResponse = async response => {
     if (response.ok && isOK) return response.json();
     throw await response.json();
   }
-  if (response.ok && isOK) return response.text();
-  throw await response.text();
```

Comment:
I believe removing this line was an accident. It should still throw if the response is not OK.